README.md

```diff
@@ -20,8 +20,8 @@ producing a log stream as its output.
 
 ## Install
 
-We don't yet distribute pre-built binaries, so to install `zq`, you must
-clone the repo and compile the source.
+To install `zq`, you can
+clone the repo and compile the source.  For Windows and MacOS, there are pre-compiled binaries at [Brim](https://www.brimsecurity.com/download/).
```

Comment:
hi @stevesmoot , good to see you here :) Thank you for the PR!

For zq, we have mac, windows, *and* linux binaries available at the zq Github release page https://github.com/brimsec/zq/releases . Would you mind updating to reflect that?

Reply:
Looks like Phil did. (Hi all!)